airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/java_connectors.py

```diff
@@ -77,6 +78,7 @@ class UnitTests(GradleTask):
     gradle_task_name = "test"
     bind_to_docker_host = True
     with_test_report = True
+    with_logs = True
```

Comment:
is there any point in maintaining both `with_logs` and `with_test_report`? seems like it should be one or the other